coreneuron/nrnoc/fadvance_core.cpp

```diff
@@ -229,6 +230,44 @@ void nrn_ba(NrnThread* nt, int bat) {
     }
 }
 
+void nrncore2nrn_send_values(NrnThread* nth) {
+  if (nrn2core_trajectory_values_ == nullptr) {
+      // standalone execution : no callbacks
+      return;
+  }
```

Comment:
@nrnhines : I added above. Otherwise standalone CoreNEURON runs were segfaulting (as nrn2core_trajectory_values_ == nullptr).

Reply:
I agree. This was an oversight on my part.